packages/scheduler/src/forks/SchedulerMock.js

```diff
@@ -79,6 +79,12 @@ let isFlushing: boolean = false;
 let needsPaint: boolean = false;
 let shouldYieldForPaint: boolean = false;
 
+var _disableYieldValue = false;
```

Comment:
nit: No need to preface this variable name with an underscore. Variables are private to the module anyway.